qemu/tests/cfg/rh_kselftests_vm.cfg

```diff
@@ -0,0 +1,13 @@
+- rh_kselftests_vm:
+    only RHEL
+    type = rh_kselftests_vm
+    virt_test_type = qemu
+    kernel_path = "/tmp/kernel"
+    kselftests_path = "/usr/libexec/kselftests"
+    variants:
+        - mm:
+            s390x:
+                kvm_module_parameters = 'hpage=1'
+            setup_hugepages = yes
+            tests_execution_cmd = "cd ${kselftests_path}/mm && sh run_vmtests.sh -t hugetlb"
+            whitelist = "hugetlb_fault_after_madv"
```

Comment:
Move into `s390x:`  as skip this case for s390x only if i remember it correctly.

Reply:
Done !